### ISSUE TITLE
Sort by column of type datetime

### ISSUE THREAD

**Issue by g-shon** [Alecaddd/sequeler#182]
When using the Data tab to view table data, you CANNOT sort column of data type DATETIME.

**Comment by Alecaddd** [comment]
Can you elaborate on this issue a bit more and provide some screenshot?
What happens when you click on the datetime column? It simply doesn't sort the data or do you get an error?

**Comment by wout** [comment]
I am experiencing this issue as well.

Sorting ascending:

![image](https://user-images.githubusercontent.com/107324/63530146-30c1e580-c506-11e9-9497-25ad6ee9c0e9.png)

Sorting descending:

![image](https://user-images.githubusercontent.com/107324/63530063-096b1880-c506-11e9-9113-5867d36c9f45.png)

**Comment by Alecaddd** [comment]
Yes, this is an issue related to the way `libGDA` prints data and time stamps.
I'm creating a workaround to sort via SQL query and not via `Gtk.TreeView`.
A fix will come out shortly, sorry for the inconvenience.
